benchmarks/fused_sum.dx

```diff
@@ -0,0 +1,11 @@
+n = if dex_test_mode() then 1000 else 1000000
+
+%bench "fused-sum"
+sum $ for i:(Fin n).
+  x = n_to_i64 (ordinal i)
+  x * x
+> 332833500
+>
+> fused-sum
+> Compile time: 22.192 ms
+> Run time:     5.186 us 	(based on 1 run)
```

Comment:
Could we filter out those lines in checked in files? They will change every time we update them, causing unnecessary churn in those files.

Reply:
In principle we could.  There is an advantage to keeping them, though, which is that it gives an indication of the rough performance without having to run anything or check a separate dashboard.  In particular, I positively want the rendered Levenshtein demo to include some timings.

In any case, I would like to defer filtering to a future PR.